index.js

```diff
@@ -17,36 +13,35 @@ module.exports.reporterName = 'teamcity';
  * @param {string}  reporterOptions.message
  * @return {wdioTeamcityReporter}
  */
-function WdioTeamcityReporter(baseReporter, wdioConf, reporterOptions = {}) {
-  this.baseReporter = baseReporter;
-
-  const opts = {
-    captureStandardOutput: typeof reporterOptions.captureStandardOutput === 'boolean'
-      ? reporterOptions.captureStandardOutput
-      : false,
-    flowId: typeof reporterOptions.flowId === 'boolean'
-      ? reporterOptions.flowId
-      : true,
-    name: typeof reporterOptions.message === 'string'
-      ? reporterOptions.message
-      : '[title]',
-  };
-
-  this.enableRealTimeOutput(opts);
-}
-
-inherits(WdioTeamcityReporter, EventEmitter);
-
-WdioTeamcityReporter.prototype.enableRealTimeOutput = function (opts) {
-  events.forEach(event => this.on(event, flow(buildFormatter(event, opts), output)));
-};
+class WdioTeamcityReporter extends WdioReporter {
+  constructor(reporterOptions) {
+    Object.assign(reporterOptions, {
+      captureStandardOutput: typeof reporterOptions.captureStandardOutput === 'boolean'
+        ? reporterOptions.captureStandardOutput
+        : false,
+      flowId: typeof reporterOptions.flowId === 'boolean'
+        ? reporterOptions.flowId
+        : true,
+      name: typeof reporterOptions.message === 'string'
+        ? reporterOptions.message
+        : '[title]',
+      stdout: true,
+    });
+    super(reporterOptions);
+    this.enableRealTimeOutput(reporterOptions);
+  }
 
-/**
- * @param  {string} msg
- * @return {string}
- */
-function output(msg) {
-  if (msg) {
-    console.log(msg);
+  enableRealTimeOutput(opts) {
+    events.forEach(event => {
+      this[event] = (...args) =>
+        flow(buildFormatter(event, opts), msg => {
+          if (msg) {
+            this.write(msg);
```

Comment:
There's no new line appended to `this.write`, so everyone gets dumped as a giant blob with no new lines.

Can you add one via:
```this.write(`${msg}\n`)```